src/css/welcome.css

```diff
@@ -523,7 +523,69 @@ button {
   }
 }
 
+.extension-promo {
+  background: pink;
```

Comment:
remove this debug!

Reply:
done in https://github.com/brave/brave-talk/pull/174/commits/c6be8e121fa9f4f35ad23a3714d26029f1ca6b96